src/index.js

```diff
@@ -1,14 +1,27 @@
-import _ from 'lodash';
 import './style.css';
 
-function component() {
-    const element = document.createElement('div');
-  
-    // Lodash, now imported by this script
-    element.innerHTML = _.join(['Hello', 'webpack'], ' ');
-    element.classList.add('hello');
-  
-    return element;
-  }
-  
-  document.body.appendChild(component());
\ No newline at end of file
+const list = [
+  {
+    index: 0,
+    description: 'Do laundry',
+    completed: false,
+  },
+  {
+    index: 1,
+    description: 'Do laundry',
+    completed: false,
+  },
+  {
+    index: 2,
+    description: 'Do Challenge',
+    completed: false,
+  },
+
+];
+
```

Comment:
- [ ] even though there are only 3 items on the list, I get 6 displayed, can you check the bug?